### ISSUE TITLE
Learning rate / epochs search for benchmark at the paper.

### ISSUE THREAD

**Issue by LeeJunHyun** [rusty1s/pytorch_geometric#284]
## ❓ Questions & Help

Hi, at first, I really appreciate this wonderful work!
This makes it easier for many researchers to conduct their experiments.

I just want to ask you about the benchmark performance for graph classification in your paper.

Is this reported after hyper parameter search such as learning rate, the number of epochs, learning rate decay, and the things that we can adjust with `argparse` in your code?

Thanks,

**Comment by rusty1s** [comment]
The `argparse` hyperparameters are fixed and there is no tuning performed on them. If you want to tune them, you should add them as lists to [this line](https://github.com/rusty1s/pytorch_geometric/blob/master/benchmark/kernel/main.py#L54).

**Comment by LeeJunHyun** [comment]
So, if I am doing a learning rate search, there's room for performance change?

Because I want to report baseline performance with this report for homogeneous training scenario.

**Comment by rusty1s** [comment]
I do not know what you mean with "room for performance change".
If you simply want to tune the learning rate, do something like this:
```python
lrs = [0.1, 0.01, 0.001, ...]
```
and replace line 54 with:
```
for lr in lrs:
```

```

**Comment by LeeJunHyun** [comment]
yes, and I also should change line 63 like this:

```python
lr=lr
```

Thanks.
Your answer was helpful for me.

 "room for performance change" means that
I should tune baselines again If I want to report my model with learning rate search.
Because baselines performance may be different with your paper.
(The reported performance may not be best performance)

**Comment by rusty1s** [comment]
Ah okay, I see what you mean. I guess you need to do that for a fair comparison :(

**Comment by LeeJunHyun** [comment]
Thanks, I will do that!

Actually, this repo is great contribution for GNN research community!

**Comment by rusty1s** [comment]
Thank you :)
